packages/nc-gui/components/smartsheet/expanded-form/index.vue

```diff
@@ -632,24 +630,21 @@ export default {
               :data-testid="`nc-expand-col-${col.title}`"
             >
               <div class="flex items-start flex-row sm:(gap-x-6) xs:(flex-col w-full) nc-expanded-cell min-h-10">
-                <div class="w-48 xs:(w-full) mt-0.25 !h-[35px]">
+                <div class="flex-1 xs:(w-full) mt-0.25 !h-[35px]">
                   <LazySmartsheetHeaderVirtualCell
                     v-if="isVirtualCol(col)"
-                    class="nc-expanded-cell-header h-full"
+                    class="nc-expanded-cell-header h-full !max-w-80"
                     :column="col"
                   />
 
-                  <LazySmartsheetHeaderCell v-else class="nc-expanded-cell-header" :column="col" />
+                  <LazySmartsheetHeaderCell v-else class="nc-expanded-cell-header !max-w-80" :column="col" />
```

Comment:
same as above

Reply:
**fixed in below Commit**

[fix : removed unwanted ! - important](https://github.com/nocodb/nocodb/pull/7114/commits/b862466785ff4be99629740e90baa00bb3b14731)
[b862466](https://github.com/nocodb/nocodb/pull/7114/commits/b862466785ff4be99629740e90baa00bb3b14731)